### ISSUE TITLE
Gradient Descent BP

### ISSUE THREAD

**Issue by touristt** [shyash/Bug-Classifier#1]
Add Backpropagation with gradient descent in Gradient_descent.py
